docs/guidance/publishing.md

```diff
@@ -59,7 +59,7 @@ If you collect detailed data on contracting processes, refer to the [OCDS implem
 
 ### (3) Build your data, systems and processes
 
-Create an OC4IDS JSON file for each project your system has information on and use a [JSON schema validator](https://json-schema.org/implementations.html#validators) to check that the files are structurally correct against OC4IDS.
+Create an OC4IDS JSON file for each project your system has information on and use the [OC4IDS Data Review Tool](http://standard.open-contracting.org/infrastructure/review) to check that the files are structurally correct against OC4IDS.
```

Comment:
See my other comment.

Reply:
Resolved in https://github.com/open-contracting/infrastructure/pull/144/commits/f15dd288f44ed6410de5637d4103b06d3f436fc3